docs/content/docs/deployment/azure-frs.md

```diff
@@ -20,126 +17,154 @@ To connect to an Azure Fluid Relay instance you first need to create an `AzureCl
 
 ```javascript
 const config = {
-    tenantId: "myTenantId",
-    tokenProvider: new AzureFunctionTokenProvider("myAzureFunctionUrl"+"/api/GetAzureToken", { userId:
-    "UserId", userName: "Test User"}),
-    orderer: "https://myOrdererUrl",
-    storage: "https://myStorageUrl",
+  tenantId: "myTenantId",
+  tokenProvider: new AzureFunctionTokenProvider(
+    "myAzureFunctionUrl" + "/api/GetAzureToken",
+    { userId: "userId", userName: "Test User" }
+  ),
+  orderer: "https://myOrdererUrl",
+  storage: "https://myStorageUrl",
 };
 
-const client = new AzureClient(config);
+const clientProps = {
+  connection: config,
+};
+
+const client = new AzureClient(clientProps);
 ```
 
 The user object can also hold optional additional user details. For example:
 
 ```javascript
-cont userDetails: ICustomUserDetails = {
+const userDetails: ICustomUserDetails = {
   email: "xyz@outlook.com",
   address: "Redmond",
 };
 
 const config = {
-    tenantId: "myTenantId",
-    tokenProvider: new AzureFunctionTokenProvider("myAzureFunctionUrl"+"/api/GetAzureToken", { userId:
-    "UserId", userName: "Test User", additionalDetails: userDetails}),
-    orderer: "https://myOrdererUrl",
-    storage: "https://myStorageUrl",
+  tenantId: "myTenantId",
+  tokenProvider: new AzureFunctionTokenProvider(
+    "myAzureFunctionUrl" + "/api/GetAzureToken",
+    { userId: "UserId", userName: "Test User", additionalDetails: userDetails }
+  ),
+  orderer: "https://myOrdererUrl",
+  storage: "https://myStorageUrl",
 };
 ```
 
 Your Azure Function will generate the token for the given user that is signed using the tenant's secret key and returned to the client without exposing the secret itself.
 
 ## Managing containers
 
-The `AzureClient` API exposes `createContainer` and `getContainer` functions to create and get containers respectively. Both functions take in the below two properties:
+The `AzureClient` API exposes `createContainer` and `getContainer` functions to create and get containers respectively. Both functions take in a _container schema_ that defines the container data model. For the
+`getContainer` function, there is an additional parameter for the container `id` of the container you wish to
+fetch.
 
-* A *container config* that defines the ID of the container and an optional entry point for logging.
-* A *container schema* that defines the container data model.
+In the container creation scenario, you can use the following pattern:
 
 ```javascript
 const schema = {
-    name: "my-container",
-    initialObjects: {
-        /* ... */
-    },
-    dynamicObjectTypes: [ /*...*/ ],
-}
-const azureClient = new AzureClient(config);
-await azureClient.createContainer({ id: "_unique-id_" }, schema);
-const { fluidContainer, containerServices } = await azureClient.getContainer({ id: "_unique-id_" }, schema);
+  name: "my-container",
```

Comment:
name was removed from schema, right?

Reply:
yup! It's just `initialObjects` and `dynamicObjectTypes` now